### PR TITLE
Write up 0.3 changelog

### PR DESCRIPTION
**Pull request by xldenis**
r @Armael

### CODE REVIEW

**Comment by Armael** [comment]
we should perhaps merge https://github.com/creusot-rs/creusot/pull/1165 (and include it in the changelog) before the release, otherwise `master` is going to be incompatible with 0.3.0 from the very beginning

**Review by Armael — APPROVED** [review]
looks good!

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,78 @@ Creusot is currently best suited for the verification of code like data-structur
...
 
-## [0.1.1] - 30/07/2024
+### Cargo Creusot
+
+Documentation using `cargo creusot doc`  now includes functions specifications and logic function bodies (Arnaud Golfouse @arnaudgolfouse).
+
+
+### Pearlite
+
+Several minor but still important changes were made to Pearlite in this release.
```
[the remark this answers is not in the document]
Comment (xldenis) [reply CHANGELOG.md:21]: Well its the things that follow, I can clarify.

**Comment by xldenis** [comment]
> we should perhaps merge https://github.com/creusot-rs/creusot/pull/1165 (and include it in the changelog) before the release, otherwise master is going to be incompatible with 0.3.0 from the very beginning

Agreed. We should merge this and then you can add a note in your PR.

**Comment by Lysxia** [comment]
Do we need to wait for a Why3 release before releasing 0.3?

**Comment by xldenis** [comment]
> Do we need to wait for a Why3 release before releasing 0.3?

No, we release as frequently as we want (and ideally often).

**Comment by jhjourdan** [comment]
Yes, but Why3 1.7.3 is going to be released really soon. Shouldn't we wait?

**Comment by xldenis** [comment]
> Yes, but Why3 1.7.3 is going to be released really soon. Shouldn't we wait?

I don't see why we should?

**Pull request merged**